javascript-modal/main.js

```diff
@@ -0,0 +1,30 @@
+/*
+
+****************************** PSEUDOCODE ******************************
+
+- get blue button
+- get row
+- get red button
+- create function
+  - change display of row to flex
+- add event
+  - if blue button is clicked, change display of row to flex
+  - if red button is clicked, change display of row to none
+
+************************************************************************
+
+*/
+
+var $blueButton = document.querySelector('.blueButton');
+var $row = document.querySelector('.row');
+var $redButton = document.querySelector('.redButton');
+
+function showPopUp(event) {
+  $row.style.display = 'flex';
```

Comment:
You definitely want to avoid setting values on styling properties directly like this, because it ends up being inline styling (aka the `style` attribute), which completely messes up specificity if this ever needs to be overwritten. Instead of doing this, you should apply this styling to a class, and add and remove those classes in these functions to match what the current state should be.